src/main/java/chessmaster/storage/Storage.java

```diff
@@ -139,11 +144,49 @@ public ChessTile[][] loadBoard() throws ChessMasterException {
             for (int col = 0; col < ChessBoard.SIZE; col++) {
                 String chessPieceString = String.valueOf(chessRowLine.charAt(col));
                 ChessPiece initialPiece = Parser.parseChessPiece(chessPieceString, rowIndex, col);
+                //@@author onx001
```

Comment:
Would be good to extract this out to another function in this class for readability
e.g. `checkValidPieces`